cpp/include/cudf/io_types.h

```diff
@@ -23,27 +23,47 @@
  */
 #pragma once
 
+typedef enum 
+{
+  FILE_PATH,
```

Comment:
Add documentation for each enum value. Comment format should be `//< This enum value does that` (note `<` which indicates this is a doxygen comment).

Reply:
Done